suites/google/googleServiceAccountKeyTest.js

```diff
@@ -18,27 +19,27 @@ function calculateSAKeyAge(creationDate) {
 }
 
 BeforeSuite(async (google, fence, users) => {
-  console.log('cleaning up old keys from the service accounts in the dcf-integration GCP project');
-  const dcfSvcAccounts = await google.listServiceAccounts('dcf-integration');
-  // console.log('#### ##: ' + dcfSvcAccounts);
-  dcfSvcAccounts.forEach(async (svcAccount) => {
-    const saName = svcAccount.email;
+  console.log('cleaning up old keys from the user0 service account in the dcf-integration GCP project');
+  const getCredsRes = await fence.do.getUserGoogleCreds(users.user0.accessTokenHeader);
```

Comment:
this is only cleaning things up for user0, before it was cleaning up for all SAs. where do you clean up user1/user2 and other SAs (or you rely on other test suites/features to do that)?

Reply:
`googleServiceAccountKeyTest.js` only cares about keys from `user0`. So these are the only ones I'm cleaning up.

`googleDataAccessTest.js` is the one that cares about all 3.

BTW, I'm aware there's some obvious code repetition there that must be refactored / moved into an `utils / commons` function.  

There are many conjectures involving the keys' clean-up process.
I just want to start with a clean slate right before each test starts.
Maybe we could do it only once in some other script (e.g., `test_setup.js`) or maybe clean up everything in the Groovy pipeline "Post" stage? Regardless of the ideal solution, this mitigation might help us get unblocked for now so we can decide/design/implement a proper solution later.